### ISSUE TITLE
Feature Request: Add Mish Activation

### ISSUE THREAD

**Issue by digantamisra98** [PaddlePaddle/Paddle#21219]
Mish is a new novel activation function proposed in this [paper](https://arxiv.org/abs/1908.08681).
It has shown promising results so far and has been adopted in several packages including:

- [TensorFlow-Addons](https://github.com/tensorflow/addons/tree/master/tensorflow_addons/activations)
- [SpaCy (Tok2Vec Layer)](https://github.com/explosion/spaCy)
- [Thinc - SpaCy's official NLP based ML library](https://github.com/explosion/thinc/releases/tag/v7.3.0)
- [Echo AI](https://github.com/digantamisra98/Echo)
- [CNTKX - Extension of Microsoft's CNTK](https://github.com/delzac/cntkx)
- [FastAI-Dev](https://github.com/fastai/fastai_dev/blob/0f613ba3205990c83de9dba0c8798a9eec5452ce/dev/local/layers.py#L441)
- [Darknet](https://github.com/AlexeyAB/darknet/commit/bf8ea4183dc265ac17f7c9d939dc815269f0a213)
- [Yolov3](https://github.com/ultralytics/yolov3/commit/444a9f7099d4ff1aef12783704e3df9a8c3aa4b3)
- [BeeDNN - Library in C++](https://github.com/edeforas/BeeDNN)
- [Gen-EfficientNet-PyTorch](https://github.com/rwightman/gen-efficientnet-pytorch)
- [dnet](https://github.com/umangjpatel/dnet/blob/master/nn/activations.py)

All benchmarks, analysis and links to official package implementations can be found in this [repository](https://github.com/digantamisra98/Mish) 

It would be nice to have Mish as an option within the activation function group.

This is the comparison of Mish with other conventional activation functions in a SEResNet-50 for CIFAR-10:  (Better accuracy and faster than GELU)
![se50_1](https://user-images.githubusercontent.com/34192716/69002745-0de37980-091b-11ea-87da-ac8d17c79e07.png)

**Comment by parap1uie-s** [comment]
Try this:

```
    def mish(self, x):
        return x * FL.tanh(FL.softplus(x))
```

**Comment by digantamisra98** [comment]
@parap1uie-s Thanks. That's really helpful. Hoping that it can be added to Paddle.

**Comment by zhhsplendid** [comment]
Hi @digantamisra98 @parap1uie-s , thank you very much for suggesting Mish activation and providing an implementation. We encourage you to contribute to PaddlePaddle and our group will think about adding Mish activation.

**Comment by digantamisra98** [comment]
@parap1uie-s Would you like to collaborate on the PR? Kind of occupied now until 5th December. Let me know. Thanks!

**Comment by Boyan-Liu** [comment]
> @parap1uie-s Would you like to collaborate on the PR? Kind of occupied now until 5th December. Let me know. Thanks!

Thanks for your attention to PaddlePaddle. zhhsplendid has relayed your information to us. If you want to contribute a new PR, please see the information in the link: https://github.com/PaddlePaddle/FluidDoc/tree/develop/doc/fluid/advanced_usage/development/contribute_to_paddle

**Comment by chunyisong** [comment]
Also hoping mish Activation added in new Paddle version！
thanks

**Comment by digantamisra98** [comment]
@Boyan-Liu @chunyisong Will set up a PR in a couple of days.

**Comment by parap1uie-s** [comment]
> > @parap1uie-s Would you like to collaborate on the PR? Kind of occupied now until 5th December. Let me know. Thanks!
> 
> Thanks for your attention to PaddlePaddle. zhhsplendid has relayed your information to us. If you want to contribute a new PR, please see the information in the link: https://github.com/PaddlePaddle/FluidDoc/tree/develop/doc/fluid/advanced_usage/development/contribute_to_paddle

Sorry for late reply... I missed the message notification and seems there's no needs to create a new PR for mish anymore. Good job for @thebhatman
